src/main/java/stsjorbsmod/patches/PlayableCursesPatch.java

```diff
@@ -0,0 +1,26 @@
+package stsjorbsmod.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePrefixPatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpireReturn;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import stsjorbsmod.powers.CatharsisPower;
+
+public class PlayableCursesPatch {
+    @SpirePatch(
+            clz = AbstractCard.class,
+            method = "canUse"
+    )
+    public static class AbstractCard_canUse
+    {
+        @SpirePrefixPatch
+        public static SpireReturn<Boolean> patch(AbstractCard __this)
+        {
+            if (__this.type == AbstractCard.CardType.CURSE && AbstractDungeon.player.hasPower(CatharsisPower.POWER_ID)) {
```

Comment:
I think this is going to interact incorrectly with Pride (showing you as being able to play it when you don't have enough energy). Rather than unilaterally returning true here, it might be better to use an instrument patch that replaces calls to `AbstractDungeon.player.hasRelic("Blue Candle")` with `(AbstractDungeon.player.hasRelic("Blue Candle") || AbstractDungeon.player.hasPower(CatharsisPower.POWER_ID))`.

The instrument patch machinery can't see at the time of the patch that the argument to hasRelic is the constant string "Blue Candle", so you have to be a little trickier in the patch replacement; the pattern looks like:

```java
@Override
public void edit(MethodCall methodCall) throws CannotCompileException {
    if (methodCall.getClassName().equals(AbstractPlayer.class.getName()) &&
        methodCall.getMethodName().equals("hasRelic"))
    {
        methodCall.replace(String.format(
                "{ $_ = ($proceed($$) || (%1$s.isBlueCandle($1) && %1$s.hasCatharsisPower($0))); }",
                PlayableCursesPatch.class.getName()));
    }
}
```